.gitignore

```diff
@@ -54,3 +54,7 @@ build-iPhoneSimulator/
 _site/
 .sass-cache/
 .jekyll-metadata
+
+# Node
+node_modules
+!node_modules/avalanche-css
```

Comment:
Yeah I wanted to stay close to the native GitHub Pages setup—without having to trigger a CI build to install npm deps.

It does not change much things to keep all the files in version control, none of them are harmful and we don't have to think about the `.gitignore` file if we have to change the `@import` statement sometimes in the future.